vpr/SRC/route/route_timing.c

```diff
@@ -100,6 +128,8 @@ bool try_timing_driven_route(struct s_router_opts router_opts,
 	// sort so net with most sinks is first.
 	std::sort(sorted_nets.begin(), sorted_nets.end(), more_sinks_than());
 
+
+
 	timing_driven_route_structs route_structs{}; // allocs route strucs (calls default constructor)
 	// contains:
 	// float *pin_criticality; /* [1..max_pins_per_net-1] */

```

Comment:
Don't think it's necessary to list what the structure contains. Better include variable descriptions and ranges as part of struct definition.